apps/dot_ext/admin.py

```diff
@@ -35,21 +35,10 @@ class MyApplicationAdmin(admin.ModelAdmin):
 
 class MyAccessTokenAdmin(admin.ModelAdmin):
 
-    list_display = ('get_user_username', 'get_app_name', 'expires', 'scope')
+    list_display = ('user', 'application', 'expires', 'scope')
```

Comment:
This results in soring by `user.id` and `application.id` but displaying the `user.username`, `application.name`, making it looks like sorting doesn't work.